Doc/glossary.rst

```diff
@@ -203,6 +203,16 @@ Glossary
       A list of bytecode instructions can be found in the documentation for
       :ref:`the dis module <bytecodes>`.
 
+   callable
+      A callable is an object that is possible to execute, possibly with a set
+      of arguments (see :term:`argument`), with the following syntax::
+
+           callable(argument1, argument2, ...)
```

Comment:
Per [devguide](https://devguide.python.org/documentation/style-guide/#use-of-whitespace) this should probably be 3 whitespaces (unless it's otherwise to be consistent with some place else in the file).

Reply:
Thanks for the suggest. I have removed the extra spaces